web/libs/datamanager/src/stores/AppStore.js

```diff
@@ -246,13 +246,14 @@ export const AppStore = types
         if (isFF(FF_REGION_VISIBILITY_FROM_URL)) {
           const { annotation: annIDFromUrl, region: regionIDFromUrl } = History.getParams();
           if (annIDFromUrl) {
-            const lsfAnnotation = self.LSF.lsf.annotationStore.annotations.find((a) => {
+            const annotationStore = self.LSF.lsf.annotationStore;
+            const lsfAnnotation = [...annotationStore.annotations, ...annotationStore.predictions].find((a) => {
```

Comment:
is it possible for `annotationStore.annotations` or `annotationStore.predictions` to be not iterable?

Reply:
if either are undefined this could cause an issue

Reply:
This is always going to be an iterable because of the store definitions and types it has within MST.

Reply:
I did push up one change to ensure that the annotationStore exists, as that aspect could be not true. If there is an annotationStore, then by the definitions we have for AnnotationStore in Editor, this will always be an iterable. Also this definition is not to be confused with the annotationStore in DataManager, which I also saw as confusing, but they are not the same and the one in question is the one from Editor.